src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorState.kt

```diff
@@ -20,6 +23,12 @@ class WaifuMotivatorState {
 
     var isDisabledInDistractionFreeMode = true
 
+    var isIdleMotivationEnabled = true
+
+    var isIdleSoundEnabled = true
+
+    var idleTimoutInMinutes = DEFAULT_IDLE_TIMEOUT_IN_MINUTES
```

Comment:
Typo of `idleTimeoutInMinutes`

Reply:
![oh snap](https://media.giphy.com/media/48FhEMYGWji8/giphy.gif)